### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sven1103**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
...
+| Quality Control of FASTQs | FASTQC | R1 + R2 FASTQs | HTML Report
+| Quality Control - Detect Outliers (Gini Index) | http://www.morgridge.net/SinQC.html | R1 FASTQ | ugly report (PDFs + Tables)
+| Find Cell barcode whitelist (OPTIONAL) | umi_tools whitelist | R1 FASTQ | whitelist.txt |
+| Extract CB/UMIs and filter CBs | umi_tools extract | R1 + R2 FASTQs + whitelist.txt | extracted FASTQs |
+| Remove PCR duplicates | umi_tools dedup | | |
+| Map reads | STAR, Kallisto, Salmon | extracted FASTQs | BAM |
+| Assign reads to genes | featureCounts, Kallisto? | BAM + transcriptome GTF | BAM
+| Count unique reads per genes per cell | umi_tools count | BAM | Counts.txt
+
+We have to add a http://multiqc.info/ report for every step where it is necessary.
```
Comment (ewels) [inline README.md:29]: NB: To confirm, there should be a single MultiQC process which runs at the end, not one for every step. If there are optional processes then dummy channels have to be created and closed for any step that does not run. Bit of a faff, but I haven't figured out a better system yet.

Comment (subwaystation) [reply README.md:29]: When writing this first draft I had something like that in mind. What I meant with report for every step are several subreports (e.g. FASTQC, STAR, etc.) generated by one MultiQC instance. 
But you are the expert here ;)

Comment (ewels) [reply README.md:29]: So I've seen "subreports" like this in one or two other pipelines (eg. our own [CAW](https://github.com/SciLifeLab/CAW)). It works fine from a technical perspective, but I'm not sure how useful they are. In practice, I'm not sure that any of us have ever looked at them before the pipeline has finished. And once it's finished, it's much better to have a single report that has everything in it.

Comment (sven1103) [reply README.md:29]: I agree with Phil, subreports do not really make sense for the kind of reporting we have in mind Simon. At the end of the day you want to have one single report per run.

**Pull request merged**